src/components/interaction-designer/Block.vue

```diff
@@ -743,23 +754,16 @@ export default {
       transition: border-radius 200ms ease-in-out;
 
       .block-exit-tag {
```

Comment:
Would need to be renamed if we rename to `block-exit-name` as mentioned above 👌

Reply:
Done!